### ISSUE TITLE
[bug] adding a git submodule breaks everything

### ISSUE THREAD

**Issue by cvandeplas** [redcanari/canari3#48]
Creating a canari package in a git repository works.
However, once you add a submodule in the canari package (in my case in the `/resources/external` subfolder) any transform launched will fail with the following stacktrace:
```
<MaltegoMessage><MaltegoTransformExceptionMessage><Exceptions><Exception>Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/canari/maltego/runner.py", line 106, in local_transform_runner
    transform = load_object(transform_py_name)()
  File "/usr/local/lib/python3.5/dist-packages/canari/maltego/runner.py", line 52, in load_object
    module = import_module(package)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "&lt;frozen importlib._bootstrap&gt;", line 986, in _gcd_import
  File "&lt;frozen importlib._bootstrap&gt;", line 969, in _find_and_load
  File "&lt;frozen importlib._bootstrap&gt;", line 944, in _find_and_load_unlocked
  File "&lt;frozen importlib._bootstrap&gt;", line 222, in _call_with_frames_removed
  File "&lt;frozen importlib._bootstrap&gt;", line 986, in _gcd_import
  File "&lt;frozen importlib._bootstrap&gt;", line 969, in _find_and_load
  File "&lt;frozen importlib._bootstrap&gt;", line 956, in _find_and_load_unlocked
ImportError: No module named 'MISP_maltego.transforms'
</Exception></Exceptions></MaltegoTransformExceptionMessage></MaltegoMessage>
```

After further investigation the issue is with the `.git/modules` folder, as removing it seems to "resolve" the issue. (however that breaks the submodule in the git repository)

**Comment by allfro** [comment]
Interesting. I'll take a look and see why that's the case 👍

**Comment by allfro** [comment]
This commit should fix things: 36459672f59602b62fdd2aa57ba9b43ae99c3815
